code/game/objects/items/storage/pill_packets.dm

```diff
@@ -11,53 +11,69 @@
 	max_storage_space = 4
 
 /obj/item/storage/pill_bottle/packet/tricordrazine
-	name = "Tricordazine pill packet"
+	name = "tricordazine pill packet"
 	icon_state = "tric_packet"
 	desc = "This packet containts tricordazine pills. Heals all types of damage slightly. Once you take them out they don't go back in. No more than 2 pills in a short period."
-	pill_type_to_fill = /obj/item/reagent_containers/pill/tricordrazine
+	pill_type_to_fill = /obj/item/reagent_containers/pill/tricordrazine/packet
 	storage_slots = 6
 	max_storage_space = 6
 
 /obj/item/storage/pill_bottle/packet/paracetamol
-	name = "Paracematol pill packet"
+	name = "paracematol pill packet"
 	desc = "This packet containts paracetamol pills, also known as tylenol. A long lasting but minor painkiller. Once you take them out they don't go back in. No more than 4 pills in a long period."
 	pill_type_to_fill = /obj/item/reagent_containers/pill/paracetamol
 
 /obj/item/storage/pill_bottle/packet/leporazine
-	name = "Leporazine pill packet"
+	name = "leporazine pill packet"
 	desc = "This packet containts leporazine pills. Rapidly stablizes the patients internal temperature. Once you take them out they don't go back in.."
 	pill_type_to_fill = /obj/item/reagent_containers/pill/leporazine
 
 /obj/item/storage/pill_bottle/packet/russian_red
-	name = "Russian Red pill packet"
+	name = "russian red pill packet"
 	desc = "This packet containts Russian Red pills. Used for field treatment of critical cases without a medic. Once you take them out they don't go back in.."
 	pill_type_to_fill = /obj/item/reagent_containers/pill/russian_red
 
 /obj/item/storage/pill_bottle/packet/ryetalyn
-	name = "Ryetalyn pill packet"
+	name = "ryetalyn pill packet"
 	desc = "This packet containts Ryetalyn pills. Used to provide a shield against bloodstream toxins. Once you take them out they don't go back in. No more than 2 pills at once."
 	pill_type_to_fill = /obj/item/reagent_containers/pill/ryetalyn
 
+/obj/item/storage/pill_bottle/packet/dylovene
+	name = "dylovene pill packet"
+	desc = "This packet containts Ryetalyn pills. Used to provide a shield against bloodstream toxins. Once you take them out they don't go back in. No more than 2 pills at once."
+	pill_type_to_fill = /obj/item/reagent_containers/pill/dylovene
+
+/obj/item/storage/pill_bottle/packet/peridaxon
+	name = "peridaxon pill packet"
+	desc = "This packet containts Peridaxon pills. Used to alleviate organ damage. Once you take them out they don't go back in. No more than 2 pills at once."
+	pill_type_to_fill = /obj/item/reagent_containers/pill/peridaxon
+
+/obj/item/storage/pill_bottle/packet/quickclot
+	name = "quickclot pill packet"
+	desc = "This packet containts Quickclot pills. Used to alleviate internal bleeding and bleeding in general. Once you take them out they don't go back in. No more than 2 pills at once."
+	pill_type_to_fill = /obj/item/reagent_containers/pill/peridaxon
```

Comment:
Quick clot packet has peridaxon instead of quick clog.

Reply:
fixed